### PR TITLE
DREIMP-4336 add paasta status writer for Cassandra cluster

### PR DESCRIPTION
**Pull request by siadat**
```
$ paasta status -s cassandra_k8s -c norcal-devc -i generic
service: cassandra_k8s
cluster: norcal-devc
    instance: generic
    Git sha:    d7b51121 (desired)
    Cassandra cluster:
        State: Running
        Nodes: 
            IP             Available  OperationMode  Joined  Datacenter    Rack          Load       Tokens  InspectedAt
            10.93.198.33   Yes        NORMAL         Yes     uswest1-devc  uswest1cdevc  24.44 GiB  256     3.5s ago
            10.93.203.77   Yes        NORMAL         Yes     uswest1-devc  uswest1cdevc  40.78 GiB  256     3.5s ago
            10.93.211.206  Yes        NORMAL         Yes     uswest1-devc  uswest1cdevc  31.58 GiB  256     3.5s ago
            10.93.178.66   Yes        NORMAL         Yes     uswest1-devc  uswest1adevc  31.86 GiB  256     3.5s ago
            10.93.135.250  Yes        NORMAL         Yes     uswest1-devc  uswest1adevc  33.85 GiB  256     3.5s ago
            10.93.167.242  Yes        NORMAL         Yes     uswest1-devc  uswest1adevc  27.42 GiB  256     3.5s ago
```

### CODE REVIEW

File: paasta_tools/cli/cmds/status.py [hunk 1]
```diff
@@ -1743,6 +1745,134 @@ def print_tron_status(
...
+    tableErrNodes = []
+
+    for node in status.get("nodes"):
+        ip = node.get("ip")
+        err = node.get("error")
+        inspectTime = datetime.strptime(node.get("inspectTime"), "%Y-%m-%dT%H:%M:%SZ").replace(tzinfo=timezone.utc)
+        details = node.get("details")
+
+        age = (now - inspectTime).total_seconds()
+        if age > 3600*24*365:
```
Comment (Rob-Johnson) [inline paasta_tools/cli/cmds/status.py:1784]: we use `humanize` for this elsewhere in paasta

https://pypi.org/project/humanize/

File: paasta_tools/cli/cmds/status.py [hunk 2]
```diff
@@ -1743,6 +1745,134 @@ def print_tron_status(
     return 0
 
 
+def print_cassandra_status(
```
Comment (Rob-Johnson) [inline paasta_tools/cli/cmds/status.py:1748]: can you write a test to verify this looks how you'd like?

File: paasta_tools/cli/cmds/status.py (hunk 1 above)
Comment (siadat) [reply paasta_tools/cli/cmds/status.py]: [Done](https://github.com/Yelp/paasta/compare/95ed3303271facb37d68254f527ceb0463d9c222..8dc6f14f14f4b12abf836a0fc158fc5152271a50)

**Review by mks-m — APPROVED** [review]
lgtm

**Comment by siadat** [comment]
@mks-m Rebasing seems to have fixed the tests. Waiting for itests.

**Comment by siadat** [comment]
Update: `make itest` passed successfully on my devbox

**Comment by siadat** [comment]
Let me know if I need to do anything else on this PR

**Comment by Rob-Johnson** [comment]
you've got a `mypy` failure
https://github.com/Yelp/paasta/runs/4884284091?check_suite_focus=true#step:6:1137

**Pull request merged**